streams/runners/storm/layout/src/main/java/com/hortonworks/streamline/streams/layout/storm/DruidEventMapper.java

```diff
@@ -37,7 +40,31 @@ public DruidEventMapper(String eventFiledName) {
 
     @Override
     public Map<String, Object> getEvent(ITuple tuple) {
+
         StreamlineEvent event = (StreamlineEvent) tuple.getValueByField(eventFiledName);
-        return event.addFieldAndValue(DruidBeamFactoryImpl.PROCESSING_TIME, System.currentTimeMillis());
+        return new DruidEvent(event.addFieldAndValue(DruidBeamFactoryImpl.PROCESSING_TIME, System.currentTimeMillis()));
+    }
+
+    public static class DruidEvent extends ForwardingMap<String, Object> {
+
+        private final StreamlineEvent event;
+
+        public DruidEvent(StreamlineEvent event) {
+            this.event = event;
+        }
+
+        public static boolean isMap(Object map) {
+            return map instanceof Map;
+        }
+
+        @Override
+        protected Map<String, Object> delegate() {
+            return event;
+        }
+
+        @Override
+        public Object get(@Nullable Object key) {
+            return StreamlineEventUtils.getFieldValue(event, (String)key);
+        }
```

Comment:
Thanks for the suggestion.  Actually I was looking for the existing util method.  

I moved the  StreamlineRuntimeUtils to common module and renamed as StreamlineUtils. let me know if any concerns.